pages/linux/update-rc.d.md

```diff
@@ -0,0 +1,19 @@
+# update-rc.d
+
+> Install and remove System-V style init script links.
+
+- Install a service:
+
+`update-rc.d {{name}} defaults`
```

Comment:
What does "name" signify here ? Just the name of the service ? If so, how does it know what is the binary to run and all the service specific stuff ?